docs: formatting and update to consul-k8s 0.40.0 (#12256)
hashicorp/consul: website/content/docs/k8s/upgrade/index.mdx

+  enabled: true
```

> To update your deployment Configuration using Helm perform the following steps:

Reviewer: ```suggestion
To update your deployment configuration using Helm, perform the following steps.
```

1. Update your local Helm repository cache:

  ```shell-session
  $ helm repo update
  ```

> 1. List all available versions. Here we can see that the latest version of `0.40.0`.

Reviewer: ```suggestion
1. List all available versions. Here you can observe that the latest version of `0.40.0`.
```

Use "we" when recommendations are being provided, otherwise, always address the user.

   ```yaml
   global:
     image: consul:1.11.2
   ```

   </CodeBlockConfig>

1. Determine your current installed chart version. In this example, version `0.39.0` (from `consul-k8s:0.39.0`) is being used.

  ```shell-session
  $ helm list --filter consul --namespace consul
  NAME  	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART        	APP VERSION
  consul	consul   	2       	2022-02-02 21:49:45.647678 -0800 PST	deployed	consul-0.39.0	1.11.1
  ```

1. Perform a `helm upgrade`:

  ```shell-session
  $ helm upgrade consul hashicorp/consul --namespace consul --version 0.39.0 --values /path/to/my/values.yaml
  ```

>   **Before performing the upgrade, be sure you've read the other sections on this page,

Reviewer: ```suggestion
  **Before performing the upgrade, be sure you have read the other sections on this page,
```
Avoid contractions as these can confuse non-native English readers.
